### ISSUE TITLE
changing single event color

### ISSUE THREAD

**Issue by Mw1995** [maddhatter/laravel-fullcalendar#78]
can i edit the color of  the event not all calendar events ???

**Comment by cesarureno** [comment]
Yes, you need to send the background color in the 6 parameter, inside the array, like this:
`$event = \Calendar::event(
                $my_event -> title, //Event title
                $my_event -> all_day, //Full day event
                $my_event -> start, //Start time
                $my_event -> end, //End time
                $my_event -> id, //Event ID
                [
                    'backgroundColor' => $back_color,
                    'textColor' => $text_color,
                    'description' => $my_event -> message
                ]
            );`

**Comment by sanket-rebelute** [comment]
backgroundColor and color both working in the same manner, I want color for entire td or we can say for that particular date like we show color for today's date.
